### PR TITLE
채팅 보내기, 채팅 기록 조회, 채팅방 정보 조회 구현

### PR DESCRIPTION
**Pull request by younghoondoodoom**

### CODE REVIEW

File: src/main/java/com/gaethering/gaetheringserver/config/WebSocketConfig.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            .setRelayPort(61613)
+            .setSystemLogin(username)
+            .setSystemPasscode(password)
+            .setClientLogin(username)
+            .setClientPasscode(password);
+    }
+
+    @Override
+    public void registerStompEndpoints(StompEndpointRegistry registry) {
+        registry.addEndpoint("/ws-connect").withSockJS();
```
Comment (naekang) [inline src/main/java/com/gaethering/gaetheringserver/config/WebSocketConfig.java:38]: 밑에있는 코드랑 두개가 존재하는 이유는 뭔지 알 수 있을까요????

**Review by naekang — COMMENTED** [review]
진짜 수고하셨습니다!! 충돌 해결까지 파이팅!🧨

**Comment by naekang** [comment]
채팅 기능... 수고하셨습니다! 😀

File: src/main/java/com/gaethering/gaetheringserver/config/WebSocketConfig.java (hunk 1 above)
Comment (younghoondoodoom) [reply src/main/java/com/gaethering/gaetheringserver/config/WebSocketConfig.java:38]: 위에 있는 거는 프론트에서 SockJS 객체랑 같이 사용하도록 허용하는 거고 아래는 websocket만 사용할 수 있도록 설정하는 겁니다!

**Comment by naekang** [comment]
|File|Coverage [83.61%]|:green_apple:|
|:-|:-:|:-:|
|[ChatServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/79a0cd7916bee4176c191390a21a933fa3069fa0/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fservice%2FChatServiceImpl.java)|100%|:green_apple:|
|[ChatController.java](https://github.com/Gaethering/Gaethering-Server/blob/79a0cd7916bee4176c191390a21a933fa3069fa0/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fcontroller%2FChatController.java)|100%|:green_apple:|
|[ChatErrorCode.java](https://github.com/Gaethering/Gaethering-Server/blob/79a0cd7916bee4176c191390a21a933fa3069fa0/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fexception%2Ferrorcode%2FChatErrorCode.java)|100%|:green_apple:|
|[WalkingTimeInfo.java](https://github.com/Gaethering/Gaethering-Server/blob/79a0cd7916bee4176c191390a21a933fa3069fa0/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fdto%2FWalkingTimeInfo.java)|91.04%|:green_apple:|
|[PostServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/79a0cd7916bee4176c191390a21a933fa3069fa0/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fboard%2Fservice%2FPostServiceImpl.java)|89.62%|:green_apple:|
|[ChatRoomInfo.java](https://github.com/Gaethering/Gaethering-Server/blob/79a0cd7916bee4176c191390a21a933fa3069fa0/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fdto%2FChatRoomInfo.java)|88.82%|:green_apple:|
|[ChatRoomMemberInfo.java](https://github.com/Gaethering/Gaethering-Server/blob/79a0cd7916bee4176c191390a21a933fa3069fa0/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fdto%2FChatRoomMemberInfo.java)|87.21%|:green_apple:|
|[ChatRoom.java](https://github.com/Gaethering/Gaethering-Server/blob/79a0cd7916bee4176c191390a21a933fa3069fa0/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fentity%2FChatRoom.java)|85%|:green_apple:|
|[ChatMessage.java](https://github.com/Gaethering/Gaethering-Server/blob/79a0cd7916bee4176c191390a21a933fa3069fa0/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fentity%2FChatMessage.java)|75.27%|:x:|
|[ChatroomMember.java](https://github.com/Gaethering/Gaethering-Server/blob/79a0cd7916bee4176c191390a21a933fa3069fa0/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fentity%2FChatroomMember.java)|59.04%|:x:|
|[ChatMessageServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/79a0cd7916bee4176c191390a21a933fa3069fa0/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fservice%2FChatMessageServiceImpl.java)|55.63%|:x:|
|[MessageController.java](https://github.com/Gaethering/Gaethering-Server/blob/79a0cd7916bee4176c191390a21a933fa3069fa0/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fchat%2Fcontroller%2FMessageController.java)|33.33%|:x:|

|Total Project Coverage|66.95%|:x:|
|:-|:-:|:-:|

**Pull request merged**